environs/spaces/spaces.go

```diff
@@ -0,0 +1,50 @@
+// Copyright 2020 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+
+package spaces
```

Comment:
Can we just make this `space`? It irks me that there are packages named `spaces`, when we use the singular in almost all other cases. Even then is it reasonable to just add to `environs.networking`?